package.json

```diff
@@ -9,7 +9,8 @@
     "css-watch": "npm run css-build -- --watch",
     "dev": "cross-env NODE_ENV=development nodemon server/index.js --watch server",
     "build": "nuxt generate",
-    "start": "cross-env NODE_ENV=production node server/index.js",
+    "build:staging": "FIRE_ENV=bootbag-staging nuxt generate",
+    "start": "nuxt start",
```

Comment:
Just curious why the start command changed?

Reply:
The previous `npm run start` command doesn't actually work (just returns a server error once the port is opened), so I've switched it to use the `nuxt start` which is the standard nuxt command - https://nuxtjs.org/docs/get-started/commands/

Reply:
Okay cool, I didn't know it didn't work :)